fission/src/systems/scene/SceneRenderer.ts

```diff
@@ -1,22 +1,21 @@
-import * as THREE from 'three';
-import SceneObject from './SceneObject';
-import WorldSystem from '../WorldSystem';
+import * as THREE from "three"
+import SceneObject from "./SceneObject"
+import WorldSystem from "../WorldSystem"
```

Comment:
Should this be changed to the `@` syntax?

Reply:
Yeah, whatever I have that does formatting in vscode really doesn't like it. I'll fix it